### ISSUE TITLE
Spectra prediction with QCxMS

### ISSUE THREAD

**Issue by hechth** [RECETOX/training-material#10]
Describe the individual steps to make a tutorial for predicting spectra with QCxMS

**Comment by hechth** [comment]
@wee-snufkin agreed kindly to contribute to this :)

**Comment by wee-snufkin** [comment]
Yes indeed, already on my to-do list! I’m away travelling right now, but will hit the ground running as soon as I’m back. Feel free to ping me if you need anything by a specific date!
